charts/otel-cloud-stack/values.yaml

```diff
@@ -0,0 +1,2141 @@
+# Top level field indicating an override for fullname
+fullnameOverride: ""
+# Top level field indicating an override for the namespace
+namespaceOverride: ""
+# Top level field indicating an override for the kube cluster version
+kubeVersionOverride: ""
+
+# Top level field specifying the name of the cluster
+clusterName: ""
+
+# Extra environment variables to add to each collector, bridge and instrumentation
+extraEnvs: []
+
+# Top level field related to the OpenTelemetry Operator
+opentelemetry-operator:
+  # Field indicating whether the operator is enabled or not
+  enabled: false
```

Comment:
Should this be enabled by default? The goal of the chart is to install an operator/collector all in one go right?

Reply:
yep probably! for my testing it was easier to not have it by default initially, but i can change that.